processing/data_collection/gazette/spiders/mt_cuiaba.py

```diff
@@ -42,7 +42,7 @@ def parse(self, response):
             edition_date = edition["publication_date"]
 
             yield Gazette(
-                date=parse(edition_date, languages=["pt"]),
+                date=parse(edition_date),
```

Comment:
**suggestion**: This change is not related to Spidermon, so it shouldn't be here. Also when parsing a date using dateparser, it is good practice to pass the language to make the parsing more reliable.

Reply:
ok I'll add a new PR to fix this. Maybe change the language, because using "pt", dateparser were returning `None`